cmake/onnxruntime_unittests.cmake

```diff
@@ -786,7 +786,7 @@ endif()
 
 if (onnxruntime_BUILD_SHARED_LIB)
   set(onnxruntime_perf_test_libs onnx_test_runner_common onnxruntime_test_utils  onnxruntime_common re2::re2
-          onnx_test_data_proto onnx_proto ${PROTOBUF_LIB} ${GETOPT_LIB_WIDE} onnxruntime ${SYS_PATH_LIB}
+          onnx_test_data_proto onnx_proto ${PROTOBUF_LIB} ${GETOPT_LIB_WIDE} onnxruntime onnxruntime_graph onnx ${SYS_PATH_LIB}
```

Comment:
Could you please avoid adding onnxruntime_graph  here?

Reply:
It looks strange if the perf test runner needs to link to a lot of ORT static libs while it dynamic link to ORT.

Reply:
And I feel the build won't pass on Linux.

Reply:
For ort format, we no longer use graph_proto instead we use Graph, which makes this necessary, is there any specific concerns we should avoid adding onnxruntime_graph here?

Reply:
I try to make the perf test tool independent of ORT core runtime and protobuf. If it depends on these things, it can't support Tensorflow.

Reply:
You may think it is a judge between ORT and TF, it shouldn't take sides.